### PR TITLE
UCP/RNDV: Fix next min_zcopy check

### PR DESCRIPTION
**Pull request by Artemy-Mellanox**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8309/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8321/ for details (Mellanox internal link).

File: src/ucp/core/ucp_request.h [hunk 1]
```diff
@@ -141,6 +141,8 @@ struct ucp_request {
                     ucp_rkey_h           rkey;           /* key for remote send buffer */
                     ucp_lane_map_t       lanes_map;      /* used lanes map */
                     ucp_lane_index_t     lane_count;     /* number of lanes used in transaction */
+                    size_t               min_zcopy;      /* upper low boundary for zcopy
```
Comment (hoopoepg) [inline src/ucp/core/ucp_request.h]: shouldn't it be value of ucp_ep?

**Comment by yosefe** [comment]
replaced by #3065

**Pull request closed without merging**